server/chat-plugins/randombattles/winrates.ts

```diff
@@ -215,13 +215,13 @@ export const pages: Chat.PageTable = {
 		const prevMonth = new Date(new Date(`${month}-15`).getTime() - (30 * 24 * 60 * 60 * 1000)).toISOString().slice(0, 7);
 		let hasButton = false;
 		if (await FS(STATS_PATH.replace('{{MONTH}}', prevMonth)).exists()) {
-			buf += `<a class="button" href="/view-winrates-${format}--${sorter}--${prevMonth}>Previous month</button>`;
+			buf += `<a class="button" href="/view-winrates-${format}--${sorter}--${prevMonth}">Previous month</button></a>`;
```

Comment:
Shouldn't this also not have the closing button tag when there's no opening tag

Reply:
Yes, that's definitely correct.